### ISSUE TITLE
Card-proxy wiki 

### ISSUE THREAD

**Issue by sanaullah82** [vnaydionov/card-proxy#1]
Hi, 
Could you share some installation and configuration document for the card-proxy project? i would like to test it. 

Regards,
Sanaullah

**Comment by vnaydionov** [comment]
Yeah, I'm working now on this. I couple of days I'm gonna upload some docs

**Comment by sanaullah82** [comment]
Thanks, also could please let me know the possibility of HSM integration through pkcs11 with this ?
